src/rSW_Site.c

```diff
@@ -458,7 +479,7 @@ SEXP onGet_SW_SIT(void) {
 	setAttrib(swrc_flags, R_NamesSymbol, swrc_names);
 
 	PROTECT(has_swrcp = NEW_LOGICAL(1));
-	LOGICAL(has_swrcp)[0] = v->site_has_swrcp;
+	LOGICAL(has_swrcp)[0] = v->inputsProvideSWRCp;

```

Comment:
onGet_SW_SIT() -- see longer comment for onGet_SW_SWRCp()
- copy`v->depthSapric` to new slot "depthSapric"